lib/import.js

```diff
@@ -91,7 +58,7 @@ class Import {
      *
      * @param {Object} options options object
      * @param {string} options.message Human readable description of changes
-     * @param {string|Object[]} options.input String of filepath or Array containing features to import
+     * @param {Stream} options.input Stream of line-delimited geojson features to import
```

Comment:
Looks like you accept a string that is the path to a file too right?

Reply:
I'll add support for a string for the API